src/components/VmsList/Vms.js

```diff
@@ -68,12 +68,14 @@ class Vms extends React.Component {
 }
 Vms.propTypes = {
   vms: PropTypes.object.isRequired,
+  config: PropTypes.object.isRequired,
   onUpdate: PropTypes.func.isRequired,
 }
 
 export default connect(
   (state) => ({
     vms: state.vms,
+    config: state.config,
```

Comment:
`!config.get('filter')`'s use in this component really just means, "always show the pool cards (for admin users)".

Suggestion for clarity in reading the code... instead of passing in the entire `config` state, use a boolean param like `alwaysShowPoolCard`.  Then this line would just be:
```js
   alwaysShowPoolCard: !state.config.get('filter'),
```

That would make the `filter` line a lot easier to understand if you don't know/remember what `!config.get('filter')` means.